examples/postgres-azure.yaml

```diff
@@ -1,23 +1,22 @@
 apiVersion: dbaas.upbound.io/v1alpha1
-kind: SQLInstance
+kind: XSQLInstance
 metadata:
   name: configuration-dbaas-azure-postgresql
   namespace: default
```

Comment:
If you using here the XR please remove the namespace

Reply:
Same of the others

Reply:
Ah, thanks nice spot